tensorflow/core/kernels/BUILD

```diff
@@ -4403,19 +4410,26 @@ tf_cc_test(
 
 if_mkl(
     tf_kernel_library(
```

Comment:
Could you make a tf_mkl_kernel_library and pull the if_mkl inside it? Buildozer has a bug which prevents us from inspecting/changing anything inside a top-level if, so pulling the if inside the new rule would make it a lot easier on our sync infra.